apps/foundation-demo/src/app/design/molecules/backdrop/backdrop/backdrop.component.scss

```diff
@@ -0,0 +1,21 @@
+$daff-sidebar-backdrop-z-index: 3;
+
+.daff-backdrop {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  background: rgba(0,0,0,0.3);
+  z-index: $daff-sidebar-backdrop-z-index;
```

Comment:
I don't think this should be here. Can we move the z-index and positioning back into `daff-sidebar`?

Reply:
I moved the z-index out, but I think the positioning needs to stay with the width and height declarations. And those need to stay in the backdrop, so it can control when it is displayed. Moving position to the sidebar breaks the backdrop completely. Moving position, width, and height just places an invisible blanket over the whole page.